src/objects/SkinnedMesh.d.ts

```diff
@@ -5,11 +5,14 @@ import { Skeleton } from './Skeleton';
 import { Mesh } from './Mesh';
 import { BufferGeometry } from '../core/BufferGeometry';
 
-export class SkinnedMesh extends Mesh {
+export class SkinnedMesh <
```

Comment:
Is it actually needed to use <> if the super class already has it?

Reply:
I think it's necessary, yes, unless a parent class has overridden the generic types with discrete types. For example:

```ts
class A<T> {}

// Must parameterize T.
class B extends A<String> {};

// Parameterization not needed.
class C extends B {}
```

Reply:
Although, maybe not with defaults? I haven't tested those before.